psl-core/src/main/java/org/linqs/psl/application/inference/InferenceApplication.java

```diff
@@ -74,7 +77,9 @@
     protected GroundRuleStore groundRuleStore;
     protected TermStore termStore;
     protected TermGenerator termGenerator;
-    protected AtomManager atomManager;
+    protected PersistedAtomManager atomManager;
+
+    private static final Logger log = LoggerFactory.getLogger(LazyMPEInference.class);
```

Comment:
!! Wrong Class object!

Make this the very first statement in the class.
Static members should generally be before non-static ones.